apps/backend/src/modules/users/user.repository.ts

```diff
@@ -60,7 +69,17 @@ class UserRepository implements Repository {
 	}
 
 	public async findByEmail(email: string): Promise<null | UserEntity> {
-		const user = await this.userModel.query().findOne({ email });
+		const user = await this.userModel
+			.query()
+			.findOne({ email })
+			.whereNull("deletedAt")
+			.execute();
+
+		return user ? UserEntity.initialize(user) : null;
+	}
+
+	public async findByEmailCreate(email: string): Promise<null | UserEntity> {
```

Comment:
why did you rename it 🤔 
```suggestion
	public async findByEmail(email: string): Promise<null | UserEntity> {
```